src/test/java/org/openelisglobal/person/PersonServiceTest.java

```diff
@@ -43,16 +52,69 @@ public void createPerson_shouldCreateNewPerson() throws Exception {
         Assert.assertEquals(firstName, savedPerson.getFirstName());
         Assert.assertEquals(lastname, savedPerson.getLastName());
     }
+    
+    @Test
+    public void updatePerson_shouldUpdateExistingPerson() throws Exception {
+        String firstName = "Sayed";
+        String lastName = "Abdo";
+
+        Person person = createPerson(firstName, lastName);
+        String personId = personService.insert(person);
+
+        // Modify some details of the person
+        String newFirstName = "UpdateSayed";
+        person.setFirstName(newFirstName);
+
+        personService.update(person);
+
+        Person updatedPerson = personService.get(personId);
+        Assert.assertEquals(newFirstName, updatedPerson.getFirstName());
+        Assert.assertEquals(lastName, updatedPerson.getLastName());
+    }
+    
+    @Test
+    public void deletePerson_shouldDeleteExistingPerson() throws Exception {
+        String firstName = "Delete";
+        String lastName = "Me";
+        String id = "1";
+
+        // Create a person object
+        Person person = createPerson(firstName, lastName);
+        person.setSysUserId(id);
+
+        // Insert person into the database
+        String personId = personService.insert(person);
+      
+        // Ensure that the person exists in the database
+        assertNotNull("Ensure that person is exist", personService.get(personId));
+
+        // Call the delete method
+        
+        personService.delete(personId, person.getSysUserId());
+        Person deletedPerson = null;
+        try {
+            deletedPerson = personService.get(personId);
+        } catch (ObjectNotFoundException e) {
+           
+        }
```

Comment:
you're not doing anything with this catch,
catching catching `ObjectNotFoundException`

Reply:
> you're not doing anything with this catch, catching `ObjectNotFoundException`

can you check my last commit and give me feedback and thanks for your review.